internal/gardener/shoot/extender/region.go

```diff
@@ -0,0 +1,32 @@
+package extender
+
+import (
+	gardener "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	imv1 "github.com/kyma-project/infrastructure-manager/api/v1"
+)
+
+const DefaultEuAccessAWSRegion = "eu-central-1"
+
+func ExtendWithRegion(runtime imv1.Runtime, shoot *gardener.Shoot) error {
+	shoot.Spec.Region = getRegion(runtime)
+
+	return nil
+}
+
+func getRegion(runtime imv1.Runtime) string {
+	if isEuAccess(runtime.Spec.Shoot.PlatformRegion) {
+		return DefaultEuAccessAWSRegion
```

Comment:
Could we have that documented either via a simple comment as in this suggestion, or have a separate documentation similar to [this](https://github.com/kyma-project/kyma-environment-broker/blob/b22742d064b4a84731e77bf7aa2fda078e01edbf/docs/contributor/03-20-eu-access.md?plain=1#L15)? 

```suggestion
                 // Only supported EuAccess region
		return DefaultEuAccessAWSRegion
```

Reply:
Region extender was removed, as region must be entirely controlled by the KEB.